data/remnant/remnant 1 introduction.txt

```diff
@@ -1364,7 +1364,7 @@ mission "Remnant: Deep Surveillance"
 			label moved
 			`	"Ivan Haataja? He moved out weeks ago. Said something about 'moving up in this world' while dragging a pallet of filing cabinets out of the building. I can look up where he is now if you need me to."`
 			`	"I'd appreciate that," you say.`
-			`	The receptionist spends a few minutes tapping away at a keyboard. "Looks like he had his office moved to... the Deep Polytechnic Institute of Valhalla?" he says with a surprised look. "Damn, must have gotten someone's attention in the government to be moved there. Do you need directions?"`
+			`	The receptionist spends a few minutes tapping away at a keyboard. "Looks like he had his office moved to... the Deep Polytechnic Institute of Valhalla?" he says with a surprised look. "Wow, must have gotten someone's attention in the government to be moved there. Do you need directions?"`
```

Comment:
```suggestion
			`	The receptionist spends a few minutes tapping away at a keyboard. "Looks like he had his office moved to... the Deep Polytechnic Institute of Valhalla?" he says with a surprised look. "Oh wow, must have gotten someone's attention in the government to be moved there. Do you need directions?"`
```
Just a small suggestion, I think “Oh wow” adds a little more energy to it while retaining the same character. You can disregard this if you want, it’ll still sound fine just thought this would be nice.

Reply:
Also small note: Adding “he” after the comma might make this sound a little better grammatically but that’s more of a side note.